sqs/sqs.py

```
@@ -110,8 +108,15 @@ def run(self):
         logger.debug(
             "Starting poll for {} every {}s".format(options.sqs_queue_url, options.poll_period)
         )
+
         while True:
-            self.poll()
+            try:
+                self.poll()
+                health_check.Healthy = True
+                sleep(self.options.poll_period)
+            except Exception:
```

Won't this make it worse?  It won't actually exit.  I had a look at this code before, there's no exception catching, so I don't understand -- it should be exiting if there are issues (and then being restarted).

---

We'll configure the liveness and readiness probes to restart the container if health check fails.

---

But it would already do that if you just let the exception bubble up; it seems like the problem is that it isn't quitting when there is an issue?